pkg/uevent/indexer_test.go

```
@@ -64,11 +64,17 @@ func TestFilterDrivesByUEventFSUUID(t *testing.T) {
 
 	d1 := createTestDrive("test-node", "D1", "d9877501-e1b5-4bac-b73f-178b29974ed5")
 	d2 := createTestDrive("test-node", "D2", filterOnFSuuid)
-
+	var err error
 	indexer := createFakeIndexer()
-	indexer.store.Add(d1)
-	indexer.store.Add(d2)
-	filteredDrive, _ := indexer.filterDrivesByUEventFSUUID(filterOnFSuuid)
+	_ = indexer.store.Add(d1)
```

can we not ignore the error here?

---

we can , but `golint ./... ` was complaining so i added the same .